src/test/java/seedu/address/model/person/HighImportanceTest.java

```diff
@@ -0,0 +1,45 @@
+package seedu.address.model.person;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static seedu.address.testutil.Assert.assertThrows;
+
+import org.junit.jupiter.api.Test;
+
+class HighImportanceTest {
+
+    @Test
+    void valueOf_invalidBooleanString_throwsIllegalArgumentException() {
+        assertThrows(IllegalArgumentException.class, () -> HighImportance.valueOf("NOT_BOOLEAN"));
+    }
+
+    @Test
+    void isValidHighImportance() {
+        // null favourite
```

Comment:
I think you meant high importance instead of favourite here?

Reply:
Yeap that's right thanks for spotting the error! 👍